scope.go

```diff
@@ -87,7 +84,36 @@ type standardScope struct {
 	timers   map[string]Timer
 }
 
-func (s *standardScope) Report(r StatsReporter) {
+// NewScope creates a new Scope around a given stats reporter with the given prefix
+func NewScope(prefix string, tags map[string]string, reporter StatsReporter, interval time.Duration) Scope {
+	if tags == nil {
+		tags = make(map[string]string)
+	}
+
+	scope := &standardScope{
+		prefix:   prefix,
+		tags:     tags,
+		reporter: reporter,
+
+		registry: &scopeRegistry{
+			subscopes: make([]Scope, 0, 5),
```

Comment:
Five is kind of a strange number, perhaps a bigger number like 4096 or just don't allocate anything upfront at all for the default allocator?